src/tests/ftest/harness/config.py

```diff
@@ -68,3 +72,10 @@ def test_harness_config_access_points(self):
         with open(self.agent_managers[0].manager.job.temporary_file, 'r') as yaml_file:
             daos_agent_yaml = yaml.safe_load(yaml_file.read())
         self.assertEqual(sorted(daos_agent_yaml['access_points']), sorted(self.access_points))
+
+        self.log.info('Verify daos_agent.yaml exclude_fabric_ifaces')
+        expected = self.params.get('exclude_fabric_ifaces', '/run/agent_config/*')
+        self.assertNotIn(expected, (None, []), 'exclude_fabric_ifaces not set in yaml')
```

Comment:
It's just an extra sanity check, since no functional tests are using exclude_fabric_ifaces.

Reply:
I mean on line 77, you obtain the list from the test yaml. Then on line 78 you're checking that it's not None or empty. Since there's nothing in between line 77 and 78, this `assertNotIn` will always pass. Is your concern that someone will modify the test yaml without reading the python code?

Reply:
> someone will modify the test yaml without reading the python code

Possibly, or that the yaml paramater is not propagated correctly. It IS overkill :)

Reply:
Although that check doesn't account for empty string either...

Reply:
I see. Thanks for the clarification!